lib/ansible/plugins/filter/aws_kms.py

```diff
@@ -0,0 +1,70 @@
+"""
+(c) 2018, Archie Gunasekara <contact@achinthagunasekara.com>
+Module to handle encrypting and decrypting of items with KMS
+"""
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+import base64
+from ansible.errors import AnsibleError, AnsibleFilterError
+
+try:
+    import botocore
+    import aws_encryption_sdk
+    HAS_DEPENDENCIES = True
+except ImportError:
+    HAS_DEPENDENCIES = False
+
+
+def aws_kms_encrypt(plaintext, key_arn):
+    """ Encrypt with KMS """
+    if not HAS_DEPENDENCIES:
```

Comment:
Hi Archie, is it worth moving this conditional block into an internal function as it's repeated twice?

Reply:
@jitran Good point. Updated with proposed changes.